### PR TITLE
fix opt.tuiEndpoint references, and entropy reloading

### PR DESCRIPTION
**Pull request by mixmix**

### CODE REVIEW

**Review by rh0delta — APPROVED** [review]

File: src/tui.ts [hunk 1]
```diff
@@ -93,15 +93,18 @@ async function main (entropy: Entropy, choices: string[], options: EntropyTuiOpt
...
+  const currentAccount = findAccountByAddressOrName(
+    storedConfig.accounts,
+    entropy?.keyring?.accounts?.registration?.address
+  )
+  if (currentAccount && currentAccount.name !== storedConfig.selectedAccount) {
     await entropy.close()
-    entropy = await loadEntropy(storedConfig.selectedAccount, options.tuiEndpoint);
+    entropy = await loadEntropy(storedConfig.selectedAccount, options.endpoint);
   }
-
```
Comment (mixmix) [inline src/tui.ts:104]: This was the major tui slowdown. `selectedAccount` is stored as `account.name` now, not `account.address`, so this was `if (true)` meaning `loadEntropy` was always being called another time

This doubled tui startup from 2s => 4s, which felt stink!

**Comment by mixmix** [comment]
Hmm, second time I've see that in CI:

```output
   program - deploy

    broken config: 
undefined:1


SyntaxError: Unexpected end of JSON input
    at JSON.parse (<anonymous>)
    at deserialize (/home/runner/work/cli/cli/src/config/encoding.ts:10:17)
    at EventEmitter.<anonymous> (/home/runner/work/cli/cli/src/common/initializeEntropy.ts:69:23)
```

**Comment by mixmix** [comment]
Have added a commit which blocks the writing of bad config (some)

File: src/tui.ts [hunk 2]
```diff
@@ -37,14 +38,19 @@ export function entropyTuiCommand () {
...
 
   const logger = new EntropyLogger('TUI', options.endpoint)
   console.clear()
-  console.log(logo) // the Entropy logo
+  const lines = logo.split('\n')
+  const lineTime = 1800 / lines.length
+  for (const line of lines) {
+    console.log(line)
+    await promisify(setTimeout)(lineTime)
+  }
```
Comment (mixmix) [inline src/tui.ts:53]: Woops I didn't mean to commit these changes with promisify + logo logging. I'll revert after tests run

**Pull request merged**